.vsts-ci.yml

```diff
@@ -260,6 +271,8 @@ extends:
               vmImage: macOS-latest
               os: macOS
             helixTargetQueue: osx.13.arm64
+            populateInternalRuntimeVariables: true
+            runtimeSourceProperties: /p:DotNetRuntimeSourceFeed=https://ci.dot.net/internal /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
```

Comment:
I fairly certain this is necessary but was missing. This job is only ran when the checkbox for enableArm64Job is used in a manual run. So, probably no one noticed.